app/controllers/catalog_controller.rb

```diff
@@ -768,7 +771,10 @@ def ot_add
                       :description => "",
                       :content     => "",
                       :type        => ot_type,
-                      :draft       => false}}
+                      :draft       => false,
+                      :manager_id  => ''}}
+    @edit[:new][:available_managers] =
+      ManageIQ::Providers::Openstack::CloudManager.all.collect { |t| [t.name, t.id] }.sort || []
```

Comment:
Damn, my comment from a couple of days ago was lost :(

We have `OrchestrationTemplateCfn#eligible_manager_types` [here](https://github.com/ManageIQ/manageiq/blob/3d42ae360d38f048dbc2e6e79e67cb9d01666ab7/app/models/orchestration_template_cfn.rb#L33-L39)

And yes, that should be rewritten as @blomquisg described. But in `OrchestrationTemplateCfn` I guess